java/src/jmri/jmrix/loconet/LocoNetSystemConnectionMemo.java

```diff
@@ -322,6 +329,20 @@ public LnStringIOManager getStringIOManager() {
         return (LnStringIOManager) classObjectMap.computeIfAbsent(StringIOManager.class, (Class<?> c) -> new LnStringIOManager(this));
     }
 
+    public Ln7gAccyRoutesManager setLn7gAccyRoutesManager(Ln7gAccyRoutesManager ln7gaccyrm) {
+        this.ln7gAcRtm = ln7gaccyrm;
+        this.ln7gAcRtm.initContext(this);
+        return this.ln7gAcRtm;
+    }
```

Comment:
You need to check `getDisabled()` before the call to `initContext()`. Otherwise you will get a NPE if `getDisabled()` returns true, which in turn will result in `configureManagers()` will throw a NPE if `getDisabled()` returns true.
```
        if (getDisabled()) {
            return null;
        }
```

Reply:
Or even better, check that `this.ln7gAcRtm` is not null before the call to `initContext()`. I suggest that you add a comment that `this.ln7gAcRtm` is null if `getDisabled()` returns true.

Reply:
Done and 'push'ed...